control-plane/go.mod

```diff
@@ -46,6 +46,7 @@ require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/bgentry/speakeasy v0.1.0 // indirect
 	github.com/cespare/xxhash/v2 v2.1.1 // indirect
+	github.com/containernetworking/cni v1.1.1 // indirect
```

Comment:
Libcni and our cni package (for shared config struct).  Libcni has a couple extra dependencies like ginko...